src/annotator/test/sidebar-test.js

```diff
@@ -48,10 +48,11 @@ describe('Sidebar', () => {
   };
 
   const createExternalContainer = () => {
-    const externalFrame = document.createElement('div');
+    externalFrame = document.createElement('div');
```

Comment:
This assumes that the test only calls `createExternalContainer` once. That might be a reasonable assumption - but it would be worth adding a check to ensure that this assumption isn't violated. Alternatively, you could record all the elements added to the DOM in tests in an array and then remove them all in the cleanup.

Reply:
I went for your second suggestions because the array pattern was already in the test for `sidebars`. I added `containers`